cluster/senza-definition.yaml

```diff
@@ -92,6 +94,25 @@ SenzaComponents:
       Tags:
         - Key: "KubernetesCluster"
           Value: kubernetes
+  - WorkerLoadBalancer:
+      Type: Senza::WeightedDnsElasticLoadBalancer
+      NameSuffix: worker
+      MainDomain: ignore-main-{{SenzaInfo.StackVersion}}.teapot.zalan.do
+      VersionDomain: ignore-version-{{SenzaInfo.StackVersion}}.teapot.zalan.do
+      HTTPPort: 4194
+      HealthCheckPath: /healthz
+      HealthCheckPort: 4194
+      Listeners:
+        - PolicyNames: []
+          Protocol: HTTP
+          InstanceProtocol: HTTP
+          InstancePort: 4194
+          LoadBalancerPort: 4194
+      Tags:
+        - Key: "Teapot"
+          Value: instance health check
```

Comment:
@mikkeloscar no this is hardcoded in Kubernetes, that is why we use it in the AWS Tags.